onnxruntime/contrib_ops/rocm/bert/fast_gelu_impl.cu

```diff
@@ -0,0 +1,100 @@
+#include "hip/hip_runtime.h"
```

Comment:
(nit):  move this by the other includes?

Reply:
this was added by amd tool: /opt/rocm/bin/hipify-perl during conversion.